Shared/Extensions/TextBox/TextBoxBehavior.UWP.cs

```diff
@@ -1,8 +1,14 @@
-﻿#if WINDOWS_UWP
+﻿#if WINDOWS_UWP || WINDOWS10_0_18362_0_OR_GREATER
```

Comment:
this is uwp.cs, should not have WINUI inside, add #if on top so that WINUI is not targeted

Reply:
I will work on a file for WINUI, because those 2 files are dependencies in the TextBoxBehavior logic